src/components/Board/Board.container.js

```diff
@@ -278,14 +278,15 @@ export class BoardContainer extends Component {
     // if (isAndroid()) downloadImages();
   }
 
-  UNSAFE_componentWillReceiveProps(nextProps) {
+  async UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
we should not grow this function, as it is going to be deprecated. See https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops
Place your code into another function.